pkg/noderesourcetopology/filter_test.go

```diff
@@ -621,159 +639,155 @@ func TestNodeResourceTopologyMultiContainerContainerScope(t *testing.T) {
+			statusErr:   "cannot align container: cnt-3",
+		},
+		{
+			description: "[17][tier1] multi init container with good allocation, multi-containers spread across NUMAs - fit",
+			initCntReq:  []map[string]string{{cpu: "4", memory: "10G"}, {cpu: "4", memory: "10G"}, {cpu: "4", memory: "10G"}},
+			cntReq:      []map[string]string{{cpu: "20", memory: "40G"}, {cpu: "20", memory: "40G"}, {cpu: "6", memory: "10G"}},
+		},
+		{
+			description: "[18][tier1] multi init container with good allocation, multi-containers with cpu over allocation - not fit",
+			initCntReq:  []map[string]string{{cpu: "4", memory: "10G"}, {cpu: "4", memory: "10G"}, {cpu: "4", memory: "10G"}},
+			cntReq:      []map[string]string{{cpu: "20", memory: "40G"}, {cpu: "20", memory: "40G"}, {cpu: "20", memory: "10G"}},
+			statusErr:   "cannot align container: cnt-3",
+		},
+		{
+			description: "[18][tier1] multi init container with good allocation, multi-containers with memory over allocation - not fit",
+			initCntReq:  []map[string]string{{cpu: "4", memory: "10G"}, {cpu: "4", memory: "10G"}, {cpu: "4", memory: "10G"}},
+			cntReq:      []map[string]string{{cpu: "20", memory: "40G"}, {cpu: "20", memory: "40G"}, {cpu: "2", memory: "60G"}},
+			statusErr:   "cannot align container: cnt-3",
+		},
+		{
+			description: "[24][tier1] multi init container with good allocation, multi-containers with over cpu allocation - not fit",
+			initCntReq:  []map[string]string{{cpu: "30", memory: "10G"}, {cpu: "30", memory: "10G"}},
+			cntReq:      []map[string]string{{cpu: "20", memory: "40G"}, {cpu: "20", memory: "40G"}, {cpu: "20", memory: "6G"}},
+			statusErr:   "cannot align container: cnt-3",
+		},
+		{
+			description: "[24][tier1] multi init container with good allocation, multi-containers with over memory allocation - not fit",
+			initCntReq:  []map[string]string{{cpu: "30", memory: "10G"}, {cpu: "30", memory: "10G"}},
+			cntReq:      []map[string]string{{cpu: "20", memory: "40G"}, {cpu: "20", memory: "40G"}, {cpu: "2", memory: "60G"}},
+			statusErr:   "cannot align container: cnt-3",
+		},
+		{
+			description: "[27][tier1] multi init container with good allocation, container with cpu over allocation - not fit",
+			initCntReq:  []map[string]string{{cpu: "30", memory: "10G"}, {cpu: "30", memory: "10G"}},
+			cntReq:      []map[string]string{{cpu: "35", memory: "40G"}},
+			statusErr:   "cannot align container: cnt-1",
+		},
+		{
+			description: "[28][tier1] multi init container with good allocation, multi-containers with good allocation - fit",
+			initCntReq:  []map[string]string{{cpu: "30", memory: "10G"}, {cpu: "30", memory: "10G"}},
+			cntReq:      []map[string]string{{cpu: "20", memory: "40G"}, {cpu: "20", memory: "40G"}},
+		},
+		{
+			description: "[29][tier1] multi init container with over cpu allocation in total, multi-containers with good allocation - fit",
```

Comment:
could you please clarify what "in total" means here?

Reply:
This is why we have the ```TESTS.md``` for, but yes I'll rephrase it